postmodern.asd

```diff
@@ -48,9 +49,9 @@
   :components
   ((:module "postmodern/tests"
             :components ((:file "test-package")
-			 (:file "tests")
-                         (:file "test-dao" :depends-on ("test-package"))
-                         (:file "test-execute-file"))))
+                         (:file "tests")
+                         #+postmodern-use-mop (:file "test-dao" :depends-on ("test-package"))
```

Comment:
Is it possible to use ASDF's `(:feature ...)` instead of an explicit reader conditional?

Reply:
Agree